### PR TITLE
cleaned up zone file examples to be more consistent

### PR DESCRIPTION
**Pull request by moonshiner**
small editorial edits to make the zone fragment examples more consistent across the document.

### CODE REVIEW

File: draft-ietf-dnsop-svcb-https.md [hunk 1]
```diff
@@ -723,12 +723,13 @@ TargetName to be a domain for which the client will have already issued
 address queries (see {{client-behavior}}).  For foo://foo.example.com:8080,
 this might look like:
 
-    ; Origin zone
+    $ORIGIN example.com.        ; Origin
     foo.example.com.            3600 IN CNAME foosvc.example.net.
     _8080._foo.foo.example.com. 3600 IN CNAME foosvc.example.net.
```
Comment (bemasc) [inline draft-ietf-dnsop-svcb-https.md:728]:
```suggestion
    $ORIGIN example.com. ; Origin
    foo                  3600 IN CNAME foosvc.example.net.
    _8080._foo.foo       3600 IN CNAME foosvc.example.net.
```

File: draft-ietf-dnsop-svcb-https.md [hunk 2]
```diff
@@ -723,12 +723,13 @@ TargetName to be a domain for which the client will have already issued
...
-    ; Service provider zone
-    foosvc.example.net. 3600 IN SVCB 1 . key65333=...
-    foosvc.example.net.  300 IN AAAA 2001:db8::1
+
+    $ORIGIN example.net.        ; Service provider zone
+    foosvc.example.net.         3600 IN SVCB 1 . key65333=...
+    foosvc.example.net.         300  IN AAAA 2001:db8::1
```
Comment (bemasc) [inline draft-ietf-dnsop-svcb-https.md:732]:
```suggestion
    $ORIGIN example.net. ; Service provider zone
    foosvc               3600 IN SVCB 1 . key65333=...
    foosvc                300 IN AAAA 2001:db8::1
```

File: draft-ietf-dnsop-svcb-https.md [hunk 3]
```diff
@@ -1167,6 +1168,7 @@ absence of an active adversary.
 
 Consider a simple zone of the form:
 
+    $ORIGIN simple.example. ; Simple example zone
     simple.example. 300 IN A    192.0.2.1
                            AAAA 2001:db8::1
```
Comment (bemasc) [inline draft-ietf-dnsop-svcb-https.md:1173]:
```suggestion
    @ 300 IN A    192.0.2.1
             AAAA 2001:db8::1
```

**Review by bemasc — CHANGES_REQUESTED** [review]
Thanks.  I proposed changing the names to relative-form, for compactness and consistency with the other examples.  You might also want to change line 1177.

**Comment by moonshiner** [comment]
Relative form is fine.  I am more pedantic about using complete names, as I tend to analyze zone data for other reasons. 
I also don't trust anything.

**Review by bemasc — APPROVED** [review]

**Pull request merged**